python/pyarrow/tests/test_parquet.py

```diff
@@ -2803,24 +2803,24 @@ def _make_dataset_for_pickling(tempdir, N=100):
     pytest.param(pytest.importorskip('cloudpickle'), id='cloudpickle')
 ])
 def test_pickle_dataset(tempdir, datadir, pickler):
-    def is_pickleable(obj):
+    def is_picklable(obj):
```

Comment:
I think that "pickle"-able is valid.

Reply:
I see. I am neutral on this. I just followed [this API](https://jsonpickle.github.io/api.html#jsonpickle.util.is_picklable).   
I have just reverted these changes.

Reply:
OK. I'll merge this pull request without these changes for now.
If some Python experts say "picklable" is better, we will use "picklable".

Reply:
The Python documentation uses "picklable" https://docs.python.org/3/library/pickle.html

Reply:
Oh... Sorry for my wrong comment...